src/main/java/ch/njol/skript/events/SimpleEvents.java

```diff
@@ -499,14 +499,8 @@ public class SimpleEvents {
 							"		send \"Hello!\"")
 					.since("2.3");
 		}
-		if (Skript.classExists("com.destroystokyo.paper.event.player.PlayerJumpEvent")) {
-			Skript.registerEvent("Jump", SimpleEvent.class, PlayerJumpEvent.class, "[player] jump[ing]")
-					.description("Called whenever a player jumps.",
-							"This event requires PaperSpigot.")
-					.examples("on jump:",
-							"	event-player does not have permission \"jump\"",
-							"	cancel event")
-					.since("2.3");
+		if ((Skript.classExists("com.destroystokyo.paper.event.player.PlayerJumpEvent")) && !(Skript.classExists("ch.njol.skript.events.EvtJump"))) {
```

Comment:
Your EvtJump class will always be registered, so this condition will never pass.

Reply:
EvtJump will always be in the classpath so it's always going to be true youre right

Reply:
to be fair i checked at the end what version this event was added in, this can be removed altogether

Reply:
I'm not sure what you mean by that. It'd be better to choose which one to register in EvtJump.class. Prioritize the Paper event if you feel like the paper event is worth keeping, otherwise just remove the paper event entirely in favor of using the statistics method.